web/site/nuxt.config.ts

```diff
@@ -3,20 +3,27 @@ export default defineNuxtConfig({
   devtools: { enabled: false },
   ssr: true,
   nitro: {
+    routeRules: {
+      '/en-CA/sbc/**': { redirect: '/en-CA' }
+    },
     prerender: {
-      routes: [
-        '/en-CA/sbc/tos'
+      routes: [],
+      ignore: [
+        '/en-CA/sbc/dashboard',
+        '/en-CA/sbc/tos',
+        '/en-CA/sbc/auth/login',
```

Comment:
this is still prerendering the login page for some reason, so i added the redirect above